src/firefly/js/ui/FieldGroupConnector.jsx

```diff
@@ -1,194 +1,175 @@
-import React, {PureComponent} from 'react';
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import React, {useContext, useState, useEffect, useRef} from 'react';
```

Comment:
Looks like no React components are created in this file any more. Should we keep it jsx?

Reply:
Yes, I think it can be removed.  I think it is required if you use JSX but in the case it does not render anything.